docs/topics/documenting-your-api.md

```diff
@@ -12,23 +12,31 @@ The most common way to document Web APIs today is to produce documentation that
 
 ---
 
+#### DRF Docs
+
+[DRF Docs][drfdocs-repo] allows you to document Web APIs made with Django REST Framework and it is authored by Emmanouil Konstantinidis. It's made to work out of the box and its setup should not take more than a couple of minutes. Complete documentation can be found on the [website][drfdocs-website] while there is also a [demo][drfdocs-demo] available for people to see what it looks like.
+
+Features include customizing the template with your branding, settings for hiding the docs depending on the environment and more.
+
+![Screenshot - DRF docs][image-drf-docs]
+
+---
+
 #### Django REST Swagger
 
 Marc Gibbons' [Django REST Swagger][django-rest-swagger] integrates REST framework with the [Swagger][swagger] API documentation tool.  The package produces well presented API documentation, and includes interactive tools for testing API endpoints.
 
-The package is fully documented, well supported, and comes highly recommended.
```

Comment:
I'd like to see "Both this package and DRF docs are..." and have the snippet listed against both packages.